src/daomop/viewer.py

```diff
@@ -49,42 +33,111 @@ def build_gui(self, container):
 
         vbox.add_widget(self.readout, stretch=0)
 
+        self.set_callback('cursor-changed', self.motion_cb)
+
+        self.top_box = Widgets.TextArea(wrap=True, editable=True)
+
         load_candidates = Widgets.TextEntry()
-        load_candidates.add_callback('load_candidates', lambda  x: self.load_candiates(x))
-        self.add_callback()
+        load_candidates.add_callback('activated', lambda x: self.load_candidates(x))
 
-        # self.set_callback('none-move', self.motion_cb)
-        self.set_callback('cursor-changed', self.motion_cb)
         accept = Widgets.Button("Accept")
-        accept.add_callback('activated', lambda x: self.onscreen_message("Accept", 1))
+        accept.add_callback('activated', lambda x: self.accept())
 
         reject = Widgets.Button("Reject")
-        reject.add_callback('activated', lambda x: self.onscreen_message("Reject", 1))
+        reject.add_callback('activated', lambda x: self.reject())
 
-        wclear = Widgets.Button("Clear")
-        wclear.add_callback('activated', lambda x: self.clear())
+        wclear = Widgets.Button("Skip")
+        wclear.add_callback('activated', lambda x: self.jump())
 
-        vbox.add_widget(accept, stretch=1)
-        vbox.add_widget(reject, stretch=1)
-        vbox.add_widget(wclear, stretch=1)
-        vbox.add_widget(load_candidates, stretch=1)
+        hbox = Widgets.HBox()
+        h2box = Widgets.HBox()
+        h2box.add_widget(accept, stretch=0)
+        h2box.add_widget(reject, stretch=0)
+        h2box.add_widget(wclear, stretch=0)
+        h2box.add_widget(load_candidates, stretch=1)
+        h2box.set_spacing(7)
+        vbox.add_widget(h2box, stretch=1)
 
         # need to put this in an hbox with an expanding label or the
         # browser wants to resize the canvas, distorting it
-        hbox = Widgets.HBox()
         hbox.add_widget(vbox, stretch=0)
         hbox.add_widget(Widgets.Label(''), stretch=1)
-
+        hbox.add_widget(self.top_box)
         container.set_widget(hbox)
 
-    def load_candiates(self, event):
-        print(event)
-        self.candidates = candidate.CandidateSet(event.text)
+    def jump(self):
+        """
+        Load the next set of images into the viewer
+        """
+        # Still not working to call candidates.next() internally
+        # if self.candidates is not None:
+        #     self.candidates.next()
```

Comment:
This frustrated me the most because it really didn't want to work. Hitting skip would seem to fetch the next set of images but never displayed them in the viewer.